tools/ci-cdk/lib/aws-sdk-rust/canary-stack.ts

```diff
@@ -174,6 +192,15 @@ export class CanaryStack extends Stack {
             resources: [`${canaryTestMrapBucketArn}`, `${canaryTestMrapBucketArn}/object/*`],
         }));
 
+        // Allow canaries to perform operations on test express bucket
+        this.lambdaExecutionRole.addToPolicy(
+            new PolicyStatement({
+                actions: ['s3express:*'],
+                effect: Effect.ALLOW,
+                resources: ["*"],
```

Comment:
Should only be allowed to do things against the bucket created in this file.

Reply:
Updated in b45dac4